web/src/connection/api/index.tsx

```diff
@@ -62,15 +62,19 @@ export const login = async (email: string, senha: string) => {
   return await secondResponse.json();
 };
 
-export const getUser = async () => {
+export const useApi = () => {
   const {jwt, setJwt} = useContext(Context);
-  const response = await fetch('http://localhost:8090/users/me', {
-    method: 'GET',
-    headers: {
-      'Content-type': 'application/json',
-      Authorization: jwt,
-    },
-  });
+  return {
+    getUser: async () => {
+      const response = await fetch('http://localhost:8090/users/me', {
```

Comment:
You should probably add the URL of the back-end on a global variable, like this:
```typescript
const API_URL = 'http://localhost:8090';
```
And use it here instead of specifying the entire URL:
```typescript
const response = await fetch(`$API_URL/users/me`, {
```